src/components/humanVsHumanGameComponent/humanVsHumanGameComponent.spec.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import HumanComponent from './humanVsHumanGameComponent';
+import Square from '../squareComponent/squareComponent';
+import Enzyme, { mount } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+Enzyme.configure({ adapter: new Adapter() });
+
+describe('Human Component', () => {
+  it('renders the Human component without crashing', () => {
+    const wrapper = mount(<HumanComponent />);
+  });
+});
+
+describe('Text placed in the square', () => {
+  it('is a X', () => {
```

Comment:
Building off the last comment, I might make this say something like `places an X for the first move`